src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/ReductionEntity.kt

```diff
@@ -19,8 +21,9 @@ data class ReductionEntity(
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   val id: Long? = null,
 
-  @Column(name = "workload_owner_id")
-  val workloadOwnerId: Long,
+  @ManyToOne
+  @JoinColumn(name = "workload_owner_id")
+  val workloadOwner: WMTWorkloadOwnerEntity,
```

Comment:
when we update reductions we will have to go via WMTWorkloadOwnerEntity to get the teamcode and staff code of the reductions which will be updated